.github/workflows/tests.yml

```diff
@@ -244,7 +244,7 @@ jobs:
 
       - name: Get composer cache directory
         id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
 
       - name: Cache composer dependencies
         uses: actions/cache@v3
```

Comment:
This step now needs to read `$GITHUB_OUTPUT` instead of `${{ steps.composer-cache.outputs.dir }}`

Reply:
Thanks. I changed it. CI is re-running, but it takes some minutes